src/app/pages/charts/components/morrisJs/morris.componenet.ts

```diff
@@ -0,0 +1,60 @@
+import {Component, OnInit} from '@angular/core';
+
+import {LineChartData} from "../../../../theme/components/morrisLineChart/morrisLineChart.component";
+import {DonutChartData} from "../../../../theme/components/morrisDonutChart/morrisDonutChart.component";
+import {BarChartData} from "../../../../theme/components/morrisBarChart/morrisBarChart.component";
+import {AreaChartData} from "../../../../theme/components/morrisAreaChart/morrisAreaChart.component";
+import {MorrisService} from "./morris.service";
```

Comment:
Please make sure you used correct style guide (spaces around the component name).
Single quotes should be used.